Backpack/src/main/java/net/skyscanner/backpack/chip/BpkChip.kt

```diff
@@ -45,53 +45,72 @@ open class BpkChip @JvmOverloads constructor(
       this.isEnabled = !disabled
     }
 
-  private val style: BpkChipStyle
+  private var _style: BpkChipStyle
```

Comment:
renamed this as this is the internal representation

Reply:
Renamed it to BpkChipAppearance